src/aabb.cpp

```diff
@@ -30,14 +30,19 @@
 
 #include <geometric_shapes/aabb.h>
 
+#include <fcl/config.h>
+#if FCL_MAJOR_VERSION > 0 || FCL_MINOR_VERSION > 5
+#include <fcl/geometry/shape/utility.h>
+#endif
+
```

Comment:
FCL 0.5 has already disappeared from all relevant platforms. All ROS 2 Jazzy platorms from REP 2000 currently offer FCL 0.7 . So all these checks for FCL 0.5 should be removed.